__tests__/flavored-compilers/images.test.js

```diff
@@ -0,0 +1,9 @@
+import { mdast, md } from '../../index';
+
+describe('image compiler', () => {
+  it('correctly serializes an image back to markdown', () => {
+    const txt = `![alt text](/path/to/image.png)`;
+
+    expect(md(mdast(txt))).toMatch(txt);
```

Comment:
Should this be a `.toStrictEqual()` check instead?

Reply:
It could, but there's an extra new line that I don't really care about?